### PR TITLE
Navigation component

### PR DESCRIPTION
**Pull request by ojame**
open for discussion and opinions please

### CODE REVIEW

File: packages/navigation/index.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    left: 0,
+    zIndex: 99,
+  },
+};
+
+module.exports = React.createClass({
+  displayName: 'Navigation',
+
+  propTypes: {
+    blur: React.PropTypes.func,
```
Comment (grapegravity) [inline packages/navigation/index.jsx]: should this be `onBlur`?

Comment (ojame) [inline packages/navigation/index.jsx]: `onBlur` is an actual event, `blur` is not

Comment (grapegravity) [inline packages/navigation/index.jsx]: but this is a callback, right?

File: packages/navigation/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Navigation
+
+A wrapper component with a logical api that glues `Bar` and `Tray`.
+
+## Usage
+
+```
+<Navigation />
```
Comment (grapegravity) [inline packages/navigation/README.md]: This should probably show at least the required props

Comment (ojame) [inline packages/navigation/README.md]: I feel like I should remove it - i'm not a massive fan of the 'Usage'. There's no point having `barItems=[]` as a prop in Usage, it means nothing. If the user wants to take a look, open up the example?

File: packages/navigation/index.jsx (hunk 1 above)
Comment (jessta) [inline packages/navigation/index.jsx]: onBlur makes sense to me. It's acting the same as the usual onBlur event.

Comment (ojame) [inline packages/navigation/index.jsx]: not for `onBlur` though. `this.props.blur` gets called when the user clicks the overlay, so when they want to close the menu. This is losing focus to the menu, so i thought `blur` works best. There's already an event in the system called `onBlur`, so I didn't want to use it. Maybe there's something better than just `blur`?

Comment (ojame) [inline packages/navigation/index.jsx]: If general consensus is `onBlur` suits, i'm happy to change :+1:

File: packages/navigation/README.md (hunk 2 above)
Comment (grapegravity) [inline packages/navigation/README.md]: Well that’s fine, but it’s more a consistency thing right now. ¯\\\_(ツ)\_/¯

File: packages/navigation/index.jsx (hunk 1 above)
Comment (grapegravity) [inline packages/navigation/index.jsx]: If you think `onBlur` is the wrong name, maybe `blur` is too? Perhaps `onMenuClose` or `onMenuBlur` or something would be better, as they’re more descriptive?

Comment (grapegravity) [inline packages/navigation/index.jsx]: But like, it definitely seems to me to be an “event,” so following that pattern feels like the right thing to do.

Comment (ojame) [inline packages/navigation/index.jsx]: `onTrayBlur` seems to be more descriptive, but I might stick to `onBlur` unless @nelix wants to weigh in

**Pull request merged**